opm/simulators/flow/python/simulators.hpp

```diff
@@ -35,13 +35,16 @@ class BlackOilSimulator
     BlackOilSimulator( const std::string &deckFilename);
     int run();
     int step();
-    int step_init();
+    int stepInit();
+    int stepCleanup();
 
 private:
     const std::string deckFilename_;
+    bool hasRunInit_;
```

Comment:
These `hasRunxxx` methods should be initialized here with:
 ```
bool hasRunInit_ = false;
bool hasRunCleanup_ = false;
```

Reply:
OK, as I understand: The reason is that in-class initialization is better than constructor initialization since the first automatically applies to all constructors of the class?  So this works like kind of a default value for the class, and if another value is required for one constructor it can be overridden there, right?

Reply:
I don't know if there is a general consensus on this; for this particular case my reasoning is quite different: I do not like the state flags `hasRunInit_` and `hasRunCleanup_` - this functionality should in some way be "guaranteed correct" (i.e. it should be design not be possible to make the errors that these two flags are protecting against). 

So - I therefor want to give the flags "less visibility" and prepare for removal.